### DISCUSSION TITLE
Dúvida: Envio de email com sendgrid.

### DISCUSSION THREAD

**Discussion by angelosilva-dev** [balta-io/forum#342] [Tira Dúvidas] [answered]
**Curso**: Fundamentos do ASP.NET 6
**Modulo**: Configuração > **Aula**: Criando um serviço de envio de E-mails
**Detalhe seu problema:** 
Na aula citada acima, apredemos como configurar no .NET, o sendgrid para enviar emails do nosso sistema para  email de usuarios.
Porém, gostaria de saber o contrário, no sentido de o usuario está preenchendo um formulário de contato de um site, e o sendgrid 
enviar o email para a caixa de entrada do email que eu escolher, para receber os orçamentos por exmeplo.

Nas configurações da plataforma do Sendgrid, é dado a opção de configurar o email "From" e realizar a verificação no menu Sender authentication.
Esse ponto serve para  o cenário da aula, no qual o nosso sistema envia para o usuário.
Gostaria de saber como configuro, para eu, dono do site receber o email do formulário preenchido em páginas de contato em meu email?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @angelosilva-dev, como vai? 
Não sei se entendi direito, mas nesse caso não seria só colocar o from e to para o E-mail da sua empresa?

Eu faço isso no balta, configuro assim e adiciono o E-mail do solicitante em cópia ou no corpo do texto para ter esta informação e poder receber depois.

Enviar como se fosse o usuário não vai acontecer, você precisaria vincular seu servidor ao servidor de E-mail deles de alguma forma, o que seria inviável.
